### PR TITLE
docs: add PostgreSQL sync schema

### PR DESCRIPTION
**Pull request by rebelice**

### CODE REVIEW

File: content/docs/change-database/synchronize-schema.md [hunk 1]
```diff
@@ -4,7 +4,9 @@ title: Synchronize Schema
 
 <hint-block type="warning">
 
-This feature is in beta and only available for MySQL.
+This feature is in beta and only available for MySQL and PostgreSQL.
+
+The feature PostgreSQL Synchronize Schema now only supports to synchronize `Schema`, `Table`, `Index`, `Constraint` and `Sequence` objects.
```
Comment (tianzhou) [inline content/docs/change-database/synchronize-schema.md:9]:
```suggestion
Synchronize Schema for PostgreSQL currently supports synchronizing `Schema`, `Table`, `Index`, `Constraint`, and `Sequence` objects.
```

Comment (tianzhou) [reply content/docs/change-database/synchronize-schema.md:9]: How about MySQL?

**Comment by rebelice** [comment]
PTAL @tianzhou

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**